src/Sylius/Component/Resource/Model/GetIdInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Resource\Model;
+
+/**
+ * @author Joseph Bielawski <stloyd@gmail.com>
+ */
+interface GetIdInterface
```

Comment:
IdAwareInterface? I know it is more for setter than getter but I still find it fits more the purpose